python/fledge/services/core/server.py

```diff
@@ -504,15 +524,41 @@ async def installation_config(cls):
             category = 'Installation'
 
             if cls._configuration_manager is None:
-                _logger.error("No configuration manager available")
+                cls._logger.error("No configuration manager available")
             await cls._configuration_manager.create_category(category, config, 'Installation', True,
                                                              display_name='Installation')
             await cls._configuration_manager.get_category_all_items(category)
 
             cls._package_cache_manager = {"update": {"last_accessed_time": ""},
                                           "upgrade": {"last_accessed_time": ""}, "list": {"last_accessed_time": ""}}
         except Exception as ex:
-            _logger.exception(str(ex))
+            cls._logger.exception(str(ex))
+            raise
+
+    @classmethod
+    async def log_config(cls):
+        """ Get the logging level configuration """
+        try:
+            config = cls._LOGGING_DEFAULT_CONFIG
+            category = 'LOGGING'
+            description = "Logging Level of Core Server"
+            if cls._configuration_manager is None:
+                cls._logger.error("No configuration manager available.")
+            await cls._configuration_manager.create_category(category, config, description, True,
```

Comment:
if None then create object of that instead of logging error. then we are using that in next statement.